src/test/resources/mappings/inventory.json

```diff
@@ -32,6 +32,10 @@
               "id": "3cb3e0cd-ccd9-47af-a570-6f3851571dca",
               "name": "Invalid ISSN"
             },
+            {
+              "id": "000000b2-03ed-469a-8179-c1092c991222",
+              "name": "ISSN with trailing X"
+            },
```

Comment:
It's a bit confusing to have this in mocked response when requested only 2 issns identifiers. You could use any of already existed ids for test data

Reply:
Yeah, right. Fixed.